KmsClient.js

```diff
@@ -57,11 +64,17 @@ export class KmsClient {
     _assert(type, 'type', 'string');
     _assert(invocationSigner, 'invocationSigner', 'object');
 
-    const keySuite = cryptoLd.suites.get(type);
-    const contextUrl = keySuite.CONTEXT_URL;
+    let SUITE_CONTEXT;
+
+    if(type === 'Sha256HmacKey2019' || type === 'AesKeyWrappingKey2019') {
+      SUITE_CONTEXT = symmetric.get(type);
+    } else {
+      const {SUITE_CONTEXT: suitContextUrl} = cryptoLd.suites.get(type);
+      SUITE_CONTEXT = suitContextUrl;
+    }
```

Comment:
Does this work instead? If so, let's use it.

```suggestion
    let SUITE_CONTEXT = symmetric.get(type);
    if(!SUITE_CONTEXT) {
      ({SUITE_CONTEXT} = cryptoLd.suites.get(type));
    }
```

Reply:
addressed here 049942d